docs/references/nextjs/overview.mdx

```diff
@@ -29,8 +29,6 @@ Clerk continues to provide drop-in support for the Next.js Pages Router. In addi
 - [`getAuth()`](/docs/references/nextjs/get-auth)
 - [`buildClerkProps()`](/docs/references/nextjs/build-clerk-props)
 
-## Other references
-
 ### `Auth` object
```

Comment:
this wouldn't be an h3 under "Server-side helpers" as its not a server-side helper, its an object.
same goes for clerkmiddleware - it wouldnt be an h3 under "server-side helpers" because its not a helper, its a middleware.
same goes for the demo repositories heading type (h3)

Reply:
https://github.com/clerk/clerk-docs/pull/1815/commits/5f3efcf64124b237a4601b2cd5bb369e33f1e922